lang/lfortran/Portfile

```diff
@@ -49,6 +46,9 @@ patchfiles-append       patch-use-sane-install-dirs.diff \
                         patch-version.diff
 # TODO: add CMake config files.
 
+# https://github.com/lfortran/lfortran/pull/3704
+patchfiles-append       patch-stacktrace.cpp-add-limits.h-for-PATH_MAX.diff
+
 configure.python        ${prefix}/bin/python${py_ver}
```

Comment:
I've asked this probably last update as well - does it work with Python 3.12? If so, please switch.

Reply:
It will be done once I fix it.

Alternatively, of course, Macports may remove older Pythons, and just make one single version to be used by every port.

P. S. It is generally a bad practice to switch something like this just for the sake of fashion. And so far I have not seen a single demonstrable case why we need Python 3.12 here or in any other port of mine. Obviously, we cannot use Python 3.7, since that one is broken on some newer systems, but AFAIK Python 3.9+ work fine.